File: learn-scala/src/main/scala/com.greatdreams.learn.scala/MainProgram.scala
package com.greatdreams.learn.scala

import com.greatdreams.learn.scala.`implicit`.{ImplicitClassTest, ImplicitTest}
import org.slf4j.{Logger, LoggerFactory}

object MainProgram {
  val logger: Logger = LoggerFactory.getLogger(MainProgram.getClass)
  def main(args: Array[String]):Unit = {
    logger.info("Hello, scala")

    ImplicitTest.main(args)
    ImplicitClassTest.main(args)
  }
}

File: learn-jetty/src/main/java/com/greatdreams/learn/jetty/MainProgram.java
package com.greatdreams.learn.jetty;

import org.eclipse.jetty.util.Jetty;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class MainProgram {
    private static Logger logger = LoggerFactory.getLogger(MainProgram.class);

    public static void main(String [] args) {
        logger.info("Jetty version is " + Jetty.VERSION);
    }
}
